pkg/kubelet/kubelet_node_status_windows.go

```diff
@@ -0,0 +1,33 @@
+// +build windows
+
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubelet
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/pkg/kubelet/winstats"
+)
+
+func addOSSpecificLabels(n *v1.Node) error {
+	osInfo, err := winstats.GetOSInfo()
+	if err != nil {
+		return err
+	}
+	n.Labels[v1.LabelWindowsBuild] = osInfo.GetBuild()
```

Comment:
doesn't v1.LabelWindowsBuild need to be added to the list in updateDefaultLabels?